src/ontology/mondo-edit.obo

```diff
@@ -414047,12 +414047,6 @@ id: MONDO:0018279
 name: obsolete congenital muscular dystrophy without intellectual disability
 subset: ordo_disorder {source="Orphanet:370980"}
```

Comment:
please remove this orphanet subset (the new QC check will fail otherwise, see the other PR for explanation)